src/System.Private.ServiceModel/src/project.lock.json

```diff
@@ -677,50 +677,51 @@
           "lib/dotnet/System.Security.Claims.dll": {}
         }
       },
-      "System.Security.Cryptography.Algorithms/4.0.0-beta-23308": {
+      "System.Security.Cryptography.Algorithms/4.0.0-beta-23316": {
         "dependencies": {
           "System.Runtime": "4.0.20",
+          "System.Security.Cryptography.Primitives": "4.0.0-beta-23316",
+          "System.IO": "4.0.10",
+          "System.Runtime.InteropServices": "4.0.20",
           "System.Resources.ResourceManager": "4.0.0",
           "System.Runtime.Handles": "4.0.0",
-          "System.Security.Cryptography.Primitives": "4.0.0-beta-23308",
           "System.Text.Encoding.Extensions": "4.0.0",
-          "System.Text.Encoding": "4.0.0",
-          "System.IO": "4.0.10"
+          "System.Text.Encoding": "4.0.0"
         },
         "compile": {
           "ref/dotnet/System.Security.Cryptography.Algorithms.dll": {}
         },
         "runtime": {
-          "lib/DNXCore50/System.Security.Cryptography.Algorithms.dll": {}
+          "lib/dotnet/System.Security.Cryptography.Algorithms.dll": {}
         }
```

Comment:
The change to "dotnet" is interesting.  Did that happen automatically when updating the project.lock.json files?